drivers/interrupt_controller/intc_ite_it8xxx2.c

```diff
@@ -27,7 +27,7 @@ static volatile uint8_t *const reg_status[] = {
 	&ISR20, &ISR21, &ISR22, &ISR23
 };
 
-static volatile uint8_t *const reg_enable[] = {
+volatile uint8_t *const reg_enable[] = {
```

Comment:
See other comment  - leave this structure private to the INTC driver and create new routines for saving and restoring the interrupts.

Reply:
Done.